modules/setup/setup.js

```diff
@@ -182,12 +225,11 @@ class Setup {
                 return;
             }
             progressbar.update(progress);
-        }, 1000);
-        this.confirmNodeStatus();
+        }, FUNCTION_CALL_INTERVAL);
```

Comment:
Can you please provide some meaningful variable name? 
if you can't read this https://blog.usejournal.com/naming-your-variables-f9477ba002e9 or some other relevant articles to learn how to name a varialble.